metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -121,7 +125,131 @@ protected Study extractAnalysisFromStudy(StudyType studyType, Study study) {
 
     @Override
     protected String getAccessionFromStandard(ReferenceAssemblyType.STANDARD standard) {
-        return standard.getRefname();
+        String refName = standard.getRefname();
+        String accession = null;
+
+        try {
+            switch (refName) {
```

Comment:
Instead of making this effectively a linear search through the use of a switch, you can create a static (initialized just once) Map and store the mappings there. That will make every search have O(1) complexity.

I would also suggest making the search case insensitive, we have seen `grch37` or similar used many times.

Reply:
Both points fixed by @iyangar